### ISSUE TITLE
Modify a process instance by activating elements inside an existing flow scope

### ISSUE THREAD

**Issue by saig0** [camunda/zeebe#9642]
## Description

We want to go one step further and allow the activation of nested elements. The element could be embedded in an embedded or an event subprocess. The given elements should be activated in the same way as root elements, by writing an `activate` command. As a result, the process instance should continue at the given elements.

For example:
![image](https://user-images.githubusercontent.com/4305769/176378993-eee32117-692e-4f88-a3f6-9d0dcaa87b13.png)

* write an `activate` command for each activate instruction
* verify that 
  * the process instance activates the given elements
  * the process instance can complete the given elements
  * the flow scope instance is completed when the given elements and the other element instances in this scope are completed

Out of scope:
* activate elements inside non-existing flow scopes
* not supported elements (see #9729)
 
Blocked by 
- #9640
